primefaces/src/main/resources/META-INF/resources/primefaces/ajaxstatus/ajaxstatus.js

```diff
@@ -189,10 +189,11 @@ PrimeFaces.widget.AjaxStatus = PrimeFaces.widget.BaseWidget.extend({
                 break;
 
             case 'complete':
+                var isFaces = args.length === 1;
```

Comment:
Please add a comment here 

Maybe there is a more safer check possible?

Reply:
Let me investigate since they both have at least 1 arg I can look for something in arg[0] to detect JSF from PF